### DISCUSSION TITLE
How can I add wildcard domain to caprover custom domain

### DISCUSSION THREAD

**Discussion by code-flag** [caprover/caprover#2228] [Q&A]
I have multitenant app that has *.cool.mydomain.com

My caprover on *.app.mydomain.com 

I have an app rock.app.mydomain.com running on caprover

But i want any request from wildcard *.cool.mydomain.com to be redirect or load rock.spp.mydomain.com 

Caprover dashboard does not allow that currently and I don't know if adding domain: [*.cool.mydomain.com] in captain-definition file with work

**Comment by githubsaturn** [comment]
CapRover doesn't allow it because it uses HTTP challenge for SSL certs and with wildcard, you cannot use an HTTP challenge to generate SSL certs.

So instead, if you don't want SSL, you can manually edit the nginx config for your app.
